app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: list) -> None:
```

Comment:
You could consider extending your type annotation to indicate that you expect to obtain a list of `Elf` instances for `elves` here.